### DISCUSSION TITLE
Error while building from source

### DISCUSSION THREAD

**Discussion by shivakumarboga** [0xPolygon/polygon-edge#671] [Q&A] [answered]
Hi team,

We are trying to install polygon-edge by building from source but we are getting below error while executing "go build -o main.go polygon-edge"

Error:
C:\Users\pavan\polygon-edge>go build -o main.go polygon-edge
cannot find package "." in:
        C:\Users\pavan\polygon-edge\vendor\polygon-edge

![error](https://user-images.githubusercontent.com/110447814/182761001-5b62ab31-7b53-4bc6-b3bb-700cbca5e8d8.jpg)

These are env set up in our windows system:
GOPATH :  %USERPROFILE%\go  (C:\Users\pavan\go)
Path: %USERPROFILE%\go\bin    (C:\Users\pavan\go\bin)

**Comment by ZeljkoBenovic** [comment, marked as answer]
Hi @shivakumarboga ,

There is an error in our docs ( we'll fix it soon ).

In order to build from source please run 
```
go build -o polygon-edge main.go
```
